jscomp/others/js_dict.mli

```diff
@@ -78,6 +78,6 @@ val fromArray : (key * 'a) array -> 'a t
 (** [fromArray entries] creates a new dictionary using with containing each
 [(key, value)] pair in [entries] *)
 
-val map : ('a -> 'b) -> 'a t -> 'b t
+val map : ('a -> 'b [@bs]) -> 'a t -> 'b t
```

Comment:
Is it reasonable to expect that you annotate any callbacks for the function?
Would it be possible to use `bs.uncurry`? I tried to apply it, but always got a `unused attribute bs.uncurry`